test/models/activation_key_test.rb

```diff
@@ -178,7 +178,8 @@ def test_available_subscriptions
       @dev_key.stubs(:get_pools).returns(cp_pools)
       @dev_key.pools = []
 
-      assert_equal [pool_one, pool_two], @dev_key.available_subscriptions
+      assert @dev_key.available_subscriptions.include? pool_one
+      assert @dev_key.available_subscriptions.include? pool_two
```

Comment:
This could also be written using assert_includes but I don't feel strongly about it :)